setup.py

```diff
@@ -49,6 +49,7 @@
             "requests==2.19.1",
             "urllib3==1.23",
             "websocket-client==0.48.0",
+            "gevent==1.4.0",
```

Comment:
yeah, this was something I mentioned above as well.  I'm not sure what to do here:
> I'm not quite sure what to do with the "requirements" in setup.py now. I just left them as they are now (eventlet being required), but that's of course a bit wrong since it's not really required someone wanted to use it with gevent. Perhaps one could pip install nameko[eventlet] or pip install nameko[gevent], but that would mean that just installing nameko would not let you run anything (that doesn't seem to new user friendly).

In any case I can add the "extras" section

Reply:
Added the "extras section".  Still not sure what to do with the main "requirements".  What I did now was raise an error in `nameko.concurrency` on an import error of the chosen backend and tell the user how to install the backend.  wdyt?

Reply:
Sorry I missed your initial comment on this. I think raising an error in `nameko.concurrency` is exactly the way to go